flashlight/common/Defines.h

```diff
@@ -10,6 +10,8 @@
 
 #include <cstdlib>
 
+#define __forceinline __attribute__((always_inline)) inline
```

Comment:
No. Choice of always inline was made to ensure the critical code section would not include overhead of function call in case `-fno-inline` is set or the inlining limit is reached. The former is the default when not optimizing ([more](https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html)).